src/vl.js

```diff
@@ -4,7 +4,10 @@ var globals = require('./globals'),
     util = require('./util'),
     consts = require('./consts');
 
-var vl = util.merge(consts, util);
+var vl = {};  // util.merge(consts, util);
```

Comment:
Thinking more about it.  This is a bad idea. I’ll revert this part — see 536b3d5e26b7070a33d1a6906a71418eb888e2a2